### ISSUE TITLE
Guidelines for Parameter Tables 

### ISSUE THREAD

**Issue by IngridT1** [redhat-documentation/supplementary-style-guide#2]
We should have standardized parameter tables with 4 columns: Parameter, Type, Default, Description

Problem: Portal makes tables ugly. 

Are tables going to look nicer in Pantheon 2? Ask the CCSOPS team (Dawn Aly). 

Reference doc tables (these tables are unnecessarily annoying and confusing with their current format) [Corrected links]:
https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/configuration_reference/keystone#auth

https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/overcloud_parameters/compute-nova-parameters

https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/command_line_interface_reference/acl#acl_delete

From Alex McLeod on OpenStack

**Comment by leswilliams44** [comment]
@IngridT1 , Are we waiting to hear from Dawn Aly?

**Comment by IngridT1** [comment]
@leswilliams44 I was just trying to capture Alex's comment, because it's important and is related to style. Do you know how to invite Dawn Aly and Alex McLeod to join this GitHub repo? Neither one of them can comment now.

**Comment by leswilliams44** [comment]
@IngridT1, I'll see what I can do to add them.

**Comment by leswilliams44** [comment]
@IngridT1, If Dawn and Alex have a GitHub user ID, we should be able to add them.

https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/inviting-users-to-join-your-organization

**Comment by bergerhoffer** [comment]
@IngridT1 are you saying that they've tried to comment on this issue and haven't been able to? Do they have a GitHub account (as @leswilliams44 mentioned) and were they logged in?

I thought that for any GitHub repo that was open, anyone with a GitHub login should be able to comment. If they are logged into GitHub, but still can't comment, then that's an issue. We don't necessarily want to add them to the style-council team, but we can add them to the org here, if they get us their GitHub IDs: https://github.com/orgs/redhat-documentation/people

**Comment by bergerhoffer** [comment]
I just confirmed with someone else that they were able to see the Comment button on this issue, being logged into GitHub but not a part of our org/repo. My guess is that they either don't have a login or weren't logged in.

**Comment by IngridT1** [comment]
@bergerhoffer I was trying to tag them in the issue, but I couldn't. They probably have GitHub IDs. In which case, I could just send them a link and ask for comments.

**Comment by bergerhoffer** [comment]
@IngridT1 Ah okay, yeah they won't pop up by default as options if they're not part of the org. Since GitHub users are global, there are probably more than user out there with their name, so we usually confirm IDs with people before trying to add them.

**Comment by IngridT1** [comment]
@leswilliams44 @bergerhoffer I started a thread on the CCS Style Council GChat, and referred Dawn to this issue. Let's see what she says.

**Comment by IngridT1** [comment]
I've opened this as a feature request with the form that Dawn Aly suggested. It's being tracked on this spreadsheet:
https://docs.google.com/spreadsheets/d/1F0M8-biMHG6ytSV2fB0j7ReNWjHIEr9ojswF_FeyX1U/edit?usp=sharing

If we are able to improve the look of the tables in Pantheon 2, we can then make a suggestion for a standard style for parameter tables. I suggest that we keep this issue open for now and wait to see what happens with Pantheon 2.

**Comment by mcljot** [comment]
Hello! Apologies for the delay submitting my input.

My issue was with the inconsistency of information presented in parameter tables in RH documentation, not that the tables are ugly. I think the tables look great so long as we don't exceed three columns, but we have a requirement to be able to include four or five columns easily so that we can represent all relevant information about parameters in a consistent way:

* Parameter
* Value
* Description
* Type
* Default
* Required/optional

This is something that I think applies to the Supplementary Style Guide so that customers have all the information they need, and don't need to analyse multifarious table column headings to find what they're looking for, varying from document to document and product to product. Happy to be corrected about that, though, of course!

**Comment by IngridT1** [comment]
@mcljot Sorry if I misrepresented you. Your explanation here is much more coherent than what I originally posted.

**Comment by inoxx03** [comment]
@mcljot @IngridT1 @leswilliams44  Is there anything that @redhat-documentation/ccs-style-council can do to move work on this issue forward?  Are we waiting on input by anyone?

**Comment by leswilliams44** [comment]
@inoxx03, It looks like Ingrid opened a ticket for the work to be done in Pantheon 2.

I've opened this as a feature request with the form that Dawn Aly suggested. It's being tracked on this spreadsheet:
https://docs.google.com/spreadsheets/d/1F0M8-biMHG6ytSV2fB0j7ReNWjHIEr9ojswF_FeyX1U/edit?usp=sharing

If we are able to improve the look of the tables in Pantheon 2, we can then make a suggestion for a standard style for parameter tables. I suggest that we keep this issue open for now and wait to see what happens with Pantheon 2.

**Comment by Srivaralakshmi** [comment]
Hi all, are there any updates on this issue?  Just trying to see where we're stuck with this/what we can do as the next steps. Thanks!

**Comment by mportman12** [comment]
@bergerhoffer @redhat-documentation/ccs-style-council any updates on whether this issue is still relevant? Especially per the Customer Portal doc UI overhaul.

**Comment by bergerhoffer** [comment]
@IngridT1 do you think the core question for this issue is still something we should investigate (trying to standardize on columns for parameter tables), or should we close this issue?

**Comment by IngridT1** [comment]
@bergerhoffer The new portal UX does help, but this is partly about using a standard format across CCS. At least, that's how Alex McLeod originally presented it to me.

**Comment by bergerhoffer** [comment]
Thanks Ingrid!

@mcljot do you think that this issue is something still worth pursuing, or should we close this issue?

**Comment by dfitzmau** [comment]
Hi @IngridT1 . I'm just checking in with you on the status of this Issue. How is it progressing?

**Comment by IngridT1** [comment]
@dfitzmau, I'm not sure that we need to do anything here. Part of the issue is really a problem with the Customer Portal, and Jupiter (or the new IBM Toolchain?) might fix that. @bergerhoffer, do we need to publish anything before those changes com about?

**Comment by mportman12** [comment]
@bergerhoffer I vote for closing.

**Comment by bburt-rh** [comment]
@bergerhoffer +1 for closing this one.

**Comment by IngridT1** [comment]
Yes, let's close it.

**Comment by bergerhoffer** [comment]
Per above, closing this issue. We can always reopen or open a new one once our tooling catches up.
